server/models/Message.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require("mongoose");
+
+const messageSchema = new mongoose.Schema(
+  {
+    sender: {
+      type: mongoose.Types.ObjectId,
+      ref: "Profile",
+      required: [true, "A message must contain the sender"],
+    },
```

Comment:
I'm curious as to why we chose to associate users/messages with `Profile` model instead of `User` model, could you walk me through your thought process?

Reply:
I chose to associate the profile because we often use the profile's photo in both the message from the sender and in the conversations. I also did this to prevent needing to additionally query the profile collection for that information on the backend.